perfrepo-cli

```diff
@@ -157,18 +157,18 @@ class TestCLI(GenericCLI):
             try:
                 test_id = int(argv[1])
                 test = self._perf_api.test_get_by_id(test_id)
-                if test is None:
-                    test = self._perf_api.test_get_by_uid(test_id)
-                if test is None:
-                    print("Test not found.")
-                else:
-                    print(test)
             except IndexError:
                 print("No ID specified!", file=sys.stderr)
                 return -1
             except ValueError:
-                print("ID needs to be an integer!", file=sys.stderr)
-                return -1
+                test = self._perf_api.test_get_by_uid(argv[1])
```

Comment:
So we're only searching by UID if the value is not int? I think we should always try to search by UID if the search by ID fails (for whatever reason - value error or no test found...).